### PR TITLE
Updates to README

### LINKED ISSUE #1: Expand README
We should update and expand the README, in line with [the recommendations we make in Collaborative Lesson Development Training](https://carpentries.github.io/lesson-development-training/collaborating-newcomers.html#readme). We should include some or all of the following:

- lesson title
- lesson description
- link to the rendered version of the lesson
- contact information for the Maintainers
- contributing information
- credits/acknowledgements
- citation info
- a brief mention of the CC-BY license, with a link to the license file.

### PR DESCRIPTION
**Pull request by anenadic**
Updates to README - fixes #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
 # Baking Sourdough Cinnamon Rolls
 
-An example lesson teaching how to bake sourdough cinnamon rolls.
+An example lesson teaching how to bake sourdough cinnamon rolls using 
+[The Carpentries Workbench lesson template](https://carpentries.github.io/sandpaper-docs/).
 
 Generated from [_crispy-doodle_](https://github.com/carpentries/crispy-doodle).
 
+## Maitainers
```
Comment (tobyhodges) [inline README.md]:
```suggestion
## Maintainers
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+## Citation
+
+To be added.
+
+## License
+
+The material in this repository is available under the [Creative Commons Attribution
+license CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/). 
+
+For more details, check [LICENCE.md](LICENCE.md).
```
Comment (tobyhodges) [inline README.md]:
```suggestion
For more details, check [LICENSE.md](LICENSE.md).
```

I make this mistake all the time too!

File: README.md [hunk 3]
```diff
@@ -1,6 +1,40 @@
 # Baking Sourdough Cinnamon Rolls
```
Comment (tobyhodges) [inline README.md:2]:
```suggestion
# Baking Sourdough Cinnamon Rolls

[Visit the lesson website](https://tobyhodges.github.io/cuddly-waffle/)
```

I recommend adding a link to the built version of the website.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,40 @@
...
+* [Aleksandra Nenadic](https://github.com/anenadic)
+
+## Contributing
+
+We welcome contributions through pull requests and issues.  Please follow the guidelines below:
+
+- The lesson is written in American English
+- We welcome changes to the recipe, as long they remain within the scope of a sourdough recipe.
+
+Please see the [CONTRIBUTING.md](CONTRIBUTING.md) for contributing guidelines and details on how to get involved with this project.
```
Comment (tobyhodges) [inline README.md:21]: This is great, thank you @anenadic

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thanks @anenadic, I noticed a couple of typos but when those are addressed this will be ready to merge.

File: README.md (hunk 3 above)
Comment (anenadic) [reply README.md:2]: Thank @tobyhodges - forgot to add it.

**Review by tobyhodges — APPROVED** [review]
Thanks @anenadic 🙌

**Pull request merged**